projects/packages/my-jetpack/global.d.ts

```diff
@@ -455,7 +455,7 @@ interface Window {
 		isAtomic: boolean;
 		sandboxedDomain: string;
 		isDevVersion: boolean;
-		userIsAdmin: string;
+		userIsAdmin: boolean;
```

Comment:
This is actually a string
![image](https://github.com/user-attachments/assets/b4090c3c-1d51-47e4-a1be-ead876c570bd)

We could possibly change that if we wanted to 🤔 I do like booleans better for this sort of thing but currently it is a string 😄

Reply:
ah, yep thanks - will put that back :)